src/App.js

```diff
@@ -11,13 +11,21 @@ function App() {
 
  const [contacts, setContact] = useState(arrOfFive);
 
+ 
 
+  const addRandomContact = () => {
+    
+    let filterList = contactsArr.filter((newContact) => {
+      return !contacts.includes(newContact)
+      })
 
+    const randomItm = filterList[Math.floor(Math.random()* filterList.length)]
 
+    setContact(contacts.concat(randomItm))
```

Comment:
We do not want to edit original array -> even if it's adding values.
Instead, you want to grab the previous/latest value and update it:

```
setContacts(contacts => [...contacts, randomItm])
```